.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">

```

Comment:
Maybe exclude this file.

Reply:
Maybe. PHPS will keep re-creating that file for me. So I'd have to keep it in .gitignore.

However, please note that this code style is PSR-2 style, which enables a quick `Ctrl + Shift + F` code formatting. If you'd like to use it, I see no harm in keeping this file here. If not, I have no qualms about ignoring it. I myself will keep using it locally.

Reply:
I think you should probably could just amend the default set to be this one. Then, PHPS shouldn't need to re-create anything, and you can safely remove it while continuing to use the rules.

Reply:
Believe me, I've tried. PHPS insists.